### PR TITLE
I18n fix

### PR DESCRIPTION
**Pull request by cromulus**
Move the twilio number into the env.sh to be set as an environment variable, additionally updating the readme

for tests to pass, twilio account info has to be accurate and functional.

### CODE REVIEW

**Comment by cromulus** [comment]
This should be fairly ready to go. All the tests pass, don't require the network. also, the translations files should be more standardized now!

File: app/language/translations/en.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
```
Comment (aberke) [inline app/language/translations/en.yaml]: do the new en.yaml and es.yaml files make the map.yaml file unneeded?

File: run_tests.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 
 import unittest
+import vcr
```
Comment (aberke) [inline run_tests.py]: used?

File: tests/test_API.py [hunk 3]
```diff
@@ -1,13 +1,12 @@
 
 from base import *
-
-
+import vcr
```
Comment (aberke) [inline tests/test_API.py]: cool!  The name of this library + it's use case with Twilio (audio) really threw me off.  This is a neat one!

**Comment by aberke** [comment]
Can you please rebase against master after responding to my comments?  I'll set the env variables on heroku hopefully this weekend.  Funny story: I knew each time you ran the tests today since I was still getting them to my number... yeah the env change would be a good one :)

File: app/language/translations/en.yaml (hunk 1 above)
Comment (cromulus) [inline app/language/translations/en.yaml]: Yes: map.yaml is not needed

Bill Cromie
t:917.215.3576

sent on the move

File: run_tests.py (hunk 2 above)
Comment (cromulus) [inline run_tests.py]: Import VCR is used as far as I can tell..

Bill Cromie
t:917.215.3576

sent on the move

File: tests/test_API.py (hunk 3 above)
Comment (cromulus) [inline tests/test_API.py]: Yeah, i have used its ruby equivalent a lot. The idea is simple and clever.
Bill Cromie
t:917.215.3576

sent on the move

**Comment by cromulus** [comment]
Will rebase against master tomorrow...

Bill Cromie
t:917.215.3576

sent on the move

File: run_tests.py (hunk 2 above)
Comment (aberke) [inline run_tests.py]: But does it need to be imported into this file?

**Comment by aberke** [comment]
Actually I realize you continued committing to the same branch, which I already merged and then made another commit on top of... So you should probably checkout a new branch from master and cherry pick your most recent commits into your branch - otherwise a complicated commit history + merge conflict hell may ensue.

File: run_tests.py (hunk 2 above)
Comment (cromulus) [inline run_tests.py]: unclear. I'm not exactly sure how python's importing works... possibly not. Totally defer to you.

**Comment by cromulus** [comment]
Ahh. gotcha. will do.

**Comment by cromulus** [comment]
closing in favor of cherry-picked commits in PR #7

**Comment by aberke** [comment]
Thanks.  And meant branch off master since work flow is 
working branch -> master -> staging -> production

**Pull request closed without merging**